client/components/clipboard-button-input/style.scss

```diff
@@ -2,6 +2,7 @@
 
 .dops-clipboard-button-input {
 	position: relative;
+	display: block;
 
 	.dops-clipboard-button {
```

Comment:
is this nested? does it need to be? either way, your changes look good! @MichaelArestad